cookbook/README.md

```diff
@@ -0,0 +1,96 @@
+# baragon-cookbook
+
+Installs Baragon (service or agent) on a node.  Baragon is the loadbalancer management service for [HubSpot's Singularity cluster management system](https://github.com/HubSpot/Singularity).
+
+## Supported Platforms
+
+Tested on an extensive range of platforms including Ubuntu 14.04 and nothing else.
+
+## Requirements
+
+### Zookeeper
+
+Baragon depends on ZooKeeper to store internal state, and perform coordination between multiple BaragonService instances if high availability is desired. Using an existing Zookeeper cluster is totally fine if you happen to have one. Separate Baragon installations can also share the same Zookeeper cluster, provided they use different zkNamespace values.
+
+### Nginx
+
+Baragon write nginx-style config files by default.  In theory other load balancers could be used, but the tool is designed around the style of config files used by nginx.
+
+The main requirement is that each application and upstream has its own file containing all necessary configuration directives (as opposed to, say, haproxy, where all configuration is parsed in order from top to bottom from a single, monolithic file).
+
+## Attributes
+
+<table>
+  <tr>
+    <th>Key</th>
+    <th>Type</th>
+    <th>Description</th>
+    <th>Default</th>
+  </tr>
+  <tr>
+    <td><tt>[:baragon][:group_name]</tt></td>
```

Comment:
This is just to be consistent with other HubSpot cookbooks (and it's not really contrary to "Best Practices" just Seth Vargo's personal opinion ;-))